setup/flex.rst

```diff
@@ -5,20 +5,20 @@ Using Symfony Flex to Manage Symfony Applications
 
 `Symfony Flex`_ is the new way to install and manage Symfony applications. Flex
 is not a new Symfony version, but a tool that replaces and improves the
-`Symfony Installer`_.
+`Symfony Installer`_ and the `Symfony Standard Edition`_.
 
 Symfony Flex **automates the most common tasks of Symfony applications**, such
-as installing and removing bundles and other dependencies. Symfony Flex works
-for Symfony 3.3 and newer versions. Starting from Symfony 4.0, Flex will be used
-by default, but it will still be optional to use.
+as installing and removing bundles and other Composer dependencies. Symfony
+Flex works for Symfony 3.3 and newer versions. Starting from Symfony 4.0, Flex
+should be used by default, but it is optional to use.
```

Comment:
`should be used by default, but it is optional to use.`
I'd say:
`should be used by default, but it is still optional.`
To to avoid the double "used/use"